### PR TITLE
Rando eliza things

### PR DESCRIPTION
**Pull request by andrewconner**

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/shared/SharedWsMessagingController.scala [hunk 1]
```diff
@@ -263,7 +263,7 @@ class SharedWsMessagingController @Inject() (
           val numUnreadUnmutedMessages = messagingCommander.getUnreadUnmutedThreadCount(socket.userId, Some(true))
           val numUnreadUnmutedNotifications = messagingCommander.getUnreadUnmutedThreadCount(socket.userId, Some(false))
           val lastModified = notificationDeliveryCommander.setAllNotificationsReadBefore(socket.userId, messageId, numUnreadUnmutedMessages, numUnreadUnmutedNotifications)
-          socket.channel.push(Json.arr("all_notifications_visited", notifId, lastModified))
+          websocketRouter.sendToUser(socket.userId, Json.arr("all_notifications_visited", notifId, lastModified))
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/shared/SharedWsMessagingController.scala:266]: Looks like it was _only_ replying to this socket, not all sockets a user is on.

Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/shared/SharedWsMessagingController.scala:266]: @gratimax

Comment (maxov) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/controllers/shared/SharedWsMessagingController.scala:266]: There are a whole bunch of other references to `socket.channel.push` in this file, should those be updated? I'm not sure how this one is special

File: kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala [hunk 2]
```diff
@@ -85,6 +85,12 @@ class MailDiscussionReceiverActor @Inject() (
           }
         }
         inbox.close(true)
+      } catch {
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala:88]: @leogrim This is just to calm down a steady flow of exceptions for no good reason

File: kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala [hunk 3]
```diff
@@ -174,7 +180,7 @@ class MailMessageReceiverPluginImpl @Inject() (
     actor.ref ! FetchNewDiscussionReplies
   }
   override def onStart() {
-    scheduleTaskOnOneMachine(actor.system, 10 seconds, 15 seconds, actor.ref, FetchNewDiscussionReplies, FetchNewDiscussionReplies.getClass.getSimpleName)
+    scheduleTaskOnOneMachine(actor.system, 3 minutes, 55 seconds, actor.ref, FetchNewDiscussionReplies, FetchNewDiscussionReplies.getClass.getSimpleName)
```
Comment (andrewconner) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/mail/MailMessageReceiverPlugin.scala:183]: Checking the inbox every 15 seconds, when it only gets a couple messages a day... slowing it down

**Pull request merged**